### DISCUSSION TITLE
Added inline style issue

### DISCUSSION THREAD

**Discussion by Bakry2002** [igordanchenko/yet-another-react-lightbox#146] [Q&A] [answered]
hey, I have an issue whenever I open the lightbox an inline style (padding specifically) is added to some elements like body and others, and I checked and that does not occur unless I add the library.
hope to catch an answer, thank you all.

**Comment by igordanchenko** [comment, marked as answer]
Can you describe why this is an issue?

This library hides browser scrollbars when the lightbox opens. To prevent page layout shift, the library adds `padding-right` to the document body and fixed-positioned elements. 

If you want to exclude some of the fixed-positioned elements, you can mark them with `yarl__no_scroll_padding` css class. 

If you want to completely disable this behavior for some reason, you can remove the `NoScroll` module:

```js
plugins={[({ remove }) => remove("no-scroll")]}
```
---

**April 2024 update** 

Starting with v3.14.0, the `NoScroll` module functionality can be disabled through the `noScroll.disabled` prop:

```jsx
noScroll={{ disabled: true }}
```

**Reply by Moriarty47** [reply]
> If you want to exclude some of the fixed-positioned elements, you can mark them with `yarl__no_scroll_padding` css class.

Is this in the docs? I almost opened an issue for the 'mistakenly hurt' situation.

**Reply by igordanchenko** [reply]
>> If you want to exclude some of the fixed-positioned elements, you can mark them with yarl__no_scroll_padding css class.
>
> Is this in the docs? I almost opened an issue for the 'mistakenly hurt' situation.

Apparently, I never documented this escape hatch. Thanks for catching this. I'll update the docs.
